app/configurator/configurator-state.spec.tsx

```diff
@@ -646,7 +646,7 @@ describe("retainChartConfigWhenSwitchingChartType", () => {
   };
 
   it("should retain appropriate x & y fields and discard the others", () => {
-    runChecks(covid19ColumnChartConfig as ColumnConfig, xyChecks);
+    runChecks(covid19ColumnChartConfig as unknown as ColumnConfig, xyChecks);
```

Comment:
I think this should be fixed before merge ?

Reply:
Yes, will remember about that!